package.json

```diff
@@ -22,7 +22,8 @@
         "build:ts:watch": "wireit",
         "build:types": "wireit",
         "build:watch": "wireit",
-        "custom-element-json": "lerna exec --ignore \"{@spectrum-web-components/{base,bundle,clear-button,close-button,modal,iconset,shared,opacity-checkerboard,styles,custom-vars-viewer,eslint-plugin},stylelint-header,@swc-react/*,documentation,example-project-rollup,example-project-webpack,swc-templates,@types/swc}\" -- cem analyze --config ../../custom-elements-manifest.config.js --packagejson",
+        "changeset-publish": "changeset publish --message \"chore: release new versions #publish\" --force-publish",
```

Comment:
I think the commands in this script will be different for Changesets vs. what they had been for Lerna. I do not believe that `--message` and `--force-publish` are valid flags that can be passed to Changesets' `publish` command. Here's a guide to what is available via the command line: https://github.com/changesets/changesets/blob/main/docs/command-line-options.md